src/templates/video.tsx

```diff
@@ -0,0 +1,203 @@
+import { addTrailingSlashIfMissing } from '~utils/add-trailing-slash-if-missing';
+import { toDateString } from '~utils/format-dates';
+import { useSiteMetadata } from '~hooks/use-site-metadata';
+import BlogShareLinks from '../components/dev-hub/blog-share-links';
+import { lineHeight, screenSize, size } from '~components/dev-hub/theme';
+import { Video as IVideo } from '~src/interfaces/video';
+
+const VIDEO_BREADCRUMB = [
+    {
+        label: 'Home',
+        target: '/',
+    },
+    {
+        label: 'Learn',
+        target: '/learn'
+    },
+
+];
+
+const TOOLTIP_TEXT = 'Video link copied to clipboard!';
+
+const Container = styled('div')`
+    margin: 0 auto;
+    @media ${screenSize.largeAndUp} {
+        display: flex;
+        justify-content: center;
+        margin-top: ${size.xxlarge};
+    }
+`;
+
+const StyledBlogShareLinks = styled(BlogShareLinks)`
+    flex-direction: column;
+    align-items: center;
+    > * {
+        margin-top: ${size.medium};
+    }
+    @media ${screenSize.upToLarge} {
+        display: inline-flex;
+        flex-direction: row;
+        > * {
+            margin-top: 0;
+            margin-left: ${size.mediumLarge};
+        }
+    }
+`;
+
+const Content = styled('article')`
+    margin: 0 auto;
+    max-width: ${size.maxContentWidth};
+    padding-left: ${size.small};
+    padding-right: ${size.small};
+    width: 100%;
+    @media ${screenSize.largeAndUp} {
+        margin: initial;
+    }
+`;
+
+const Icons = styled('div')`
+    margin: 0 ${size.small};
+    span {
+        &:not(:first-of-type) {
+            margin-left: ${size.small};
+        }
+        padding: 0 ${size.tiny};
+    }
+    @media ${screenSize.largeAndUp} {
+        display: flex;
+        flex-direction: column;
+        margin: 0 ${size.default};
+        span:not(:first-of-type) {
+            margin-top: ${size.small};
+        }
+    }
+`;
+
+const StyledParagraph = styled(P)`
+    line-height: ${lineHeight.default};
+`;
+
+const dateFormatOptions = {
+    month: 'short',
+    day: '2-digit',
+    year: 'numeric',
+    timeZone: 'UTC',
+};
+
+type VideoProps = {
+    pageContext: {
+        data: IVideo;
+    };
+};
+
+const Video = ({
+    pageContext: {
+        data: {
+            slug,
+            description,
+            publishDate,
+            thumbnailUrl,
+            title,
+            videoId,
+            mediaType,
+        },
+    },
+}: VideoProps) => {
+    const { siteUrl } = useSiteMetadata();
+    const pageUrl = addTrailingSlashIfMissing(`${siteUrl}${slug}`);
+    const formattedPublishDate = toDateString(publishDate, dateFormatOptions);
+    const capitalizedBreadcrumb = mediaType.charAt(0).toUpperCase() + mediaType.slice(1);
+
+    const videoBreadcrumb = useMemo(
+        () => {
+            const videoBreadCrumb = [
+            ...VIDEO_BREADCRUMB,
+                {
+                    label: `${capitalizedBreadcrumb} Video`,
+                    target: `/type/${mediaType}`,
```

Comment:
I noticed on the /type/twitch page it is just `Twitch` whereas here it is `Twitch Video` for example. We should standardize to one or the other. Any ideas?

Reply:
good catch. It is Twitch Video according to wireframe.Let me change it.